source/fundamentals/auth.txt

```diff
@@ -52,16 +52,7 @@ Mechanisms
 Default
```

Comment:
I: I can't comment on the line directly, but you can remove * :ref:`MONGODB-CR <mongodb-cr-auth-mechanism>` from the list in the overview (and any other references to that ref in the docs)